restructuredtext_lint/lint.py

```diff
@@ -80,9 +75,24 @@ def error_collector(data):
     return errors
 
 
-def lint_file(filepath, encoding=None):
+def lint(content, filepath=None, ignore_sphinx=True):
```

Comment:
Instead of making this specific to Sphinx can we have something about `ignore_directives=None`, `ignore_roles=None`? Maybe we can provide a constants for people to use for Sphinx directives/roles